ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -17,81 +17,75 @@
+
+  private static final String COMMAND_CONSUMER_PREFIX  =
+      KSQL_CONFIG_PREFIX + "server.command.consumer.";
+  private static final String COMMAND_PRODUCER_PREFIX  =
+      KSQL_CONFIG_PREFIX + "server.command.producer.";
+
+  static final String STREAMED_QUERY_DISCONNECT_CHECK_MS_CONFIG =
+      "query.stream.disconnect.check";
+
+  private static final String STREAMED_QUERY_DISCONNECT_CHECK_MS_DOC =
           "How often to send an empty line as part of the response while streaming queries as "
               + "JSON; this helps proactively determine if the connection has been terminated in "
               + "order to avoid keeping the created streams job alive longer than necessary";
 
-  public static final String
-      DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_CONFIG = "ksql.server.command.response.timeout.ms";
-  public static final ConfigDef.Type
-      DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_TYPE = ConfigDef.Type.LONG;
-  public static final Long
-      DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_DEFAULT = 5000L;
-  public static final ConfigDef.Importance
-      DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_IMPORTANCE = ConfigDef.Importance.LOW;
-  public static final String
-      DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_DOC =
-          "How long to wait for a distributed command to be executed by the local node before "
+  static final String DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_CONFIG =
+      KSQL_CONFIG_PREFIX + "server.command.response.timeout.ms";
+
+  private static final String DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_DOC =
+            "How long to wait for a distributed command to be executed by the local node before "
               + "returning a response";
 
-  public static final String INSTALL_DIR_CONFIG = "ksql.server.install.dir";
-  public static final String INSTALL_DIR_DOC
+  static final String INSTALL_DIR_CONFIG = KSQL_CONFIG_PREFIX + "server.install.dir";
+  private static final String INSTALL_DIR_DOC
       = "The directory that ksql is installed in. This is set in the ksql-server-start script.";
 
-  public static final String COMMAND_TOPIC_SUFFIX = "command_topic";
-
-  public static final String KSQL_WEBSOCKETS_NUM_THREADS = "ksql.server.websockets.num.threads";
-
-  private static final int DEFAULT_WEBSOCKETS_THREADS = 5;
+  static final String COMMAND_TOPIC_SUFFIX = "command_topic";
 
+  static final String KSQL_WEBSOCKETS_NUM_THREADS =
+      KSQL_CONFIG_PREFIX + "server.websockets.num.threads";
+  private static final String KSQL_WEBSOCKETS_NUM_THREADS_DOC =
+      "The number of websocket threads to handle query results";
 
   private static final ConfigDef CONFIG_DEF;
 
   static {
     CONFIG_DEF = baseConfigDef().define(
         STREAMED_QUERY_DISCONNECT_CHECK_MS_CONFIG,
-        STREAMED_QUERY_DISCONNECT_CHECK_MS_TYPE,
-        STREAMED_QUERY_DISCONNECT_CHECK_MS_DEFAULT,
-        STREAMED_QUERY_DISCONNECT_CHECK_MS_IMPORTANCE,
+        Type.LONG,
+        1000L,
```

Comment:
shouldnt defaults be extracted?

Reply:
The Kafka one's don't

Reply:
How do I change these config values in production - i.e. if they arent suitable?

Reply:
By setting them in your config file or using a `-Dsome.setting=foo' on the command line, as you would with any config.

The reason there is generally no need to extract the default out into a constant is because other areas of the code should never need it.  The config itself holds the default.

The same could be said for the doc field, but this is normally pulled out to keep the definition chain more readable.